Dockerfile

```diff
@@ -11,4 +11,4 @@ LABEL "com.github.actions.color"="orange"
 WORKDIR /app
 COPY . .
 RUN yarn --frozen-lockfile
-ENTRYPOINT [ "/app/node_modules/.bin/probot", "receive", "/app/index.js" ]
\ No newline at end of file
+ENTRYPOINT [ "/app/node_modules/.bin/probot", "run", "/app/index.js" ]
```

Comment:
hmm, can you create a new Dockerfile for the `Dockerfile` action to use?
https://github.com/release-drafter/release-drafter/blob/8ef32da3eaa2ce71a765835e12aae337792e4492/docker/action.yml#L5

Perhaps call the file `action.Dockerfile`

Reply:
I splitted the current ENTRYPOINT into ENTRYPOINT + CMD, so the default behaviour is the same. One can overwrite it with a own command like  `run /app/index.js`. Better now?

Reply:
Yup